goals.md

```diff
@@ -3,8 +3,8 @@ layout: default
 ---
 
 <picture class="full pixels">
-    <source srcset="../assets/goals-dark.gif" media="(prefers-color-scheme: dark)">
-    <img src="../assets/goals.gif">
+    <source srcset="./assets/goals-dark.gif" media="(prefers-color-scheme: dark)">
+    <img src="./assets/goals.gif">
```

Comment:
Yes, I do see the problem with the path, but the animated GIF does seem to show up and work correctly with Firefox on Fedora:
![image](https://github.com/user-attachments/assets/feb47876-e8a7-443b-91a3-1f62878f0dfc)

Is the animation not showing up with some specific browser?

I am asking because this is the sort of confusing and non-obvious behaviour that triggers all sorts of alarms in my head.  :)